test/functional/pdo_sqlsrv/pdo_fetch_cursorBuffered_decimal_bindColumn_int.phpt

```diff
@@ -1,91 +1,96 @@
 --TEST--
 prepare with cursor buffered and fetch a decimal column with the column bound and specified to pdo type int
 --SKIPIF--
-
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-require_once("MsSetup.inc");
-$conn = new PDO( "sqlsrv:server=$server; database=$databaseName", $uid, $pwd);
-$conn->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION );
-$sample = 1234567890.1234;
-
-$query = 'CREATE TABLE #TESTTABLE (exist decimal(18, 8))';
-$stmt = $conn->exec($query);
-$query = 'INSERT INTO #TESTTABLE VALUES(:p0)';
-$stmt = $conn->prepare($query);
-$stmt->bindValue(':p0', $sample, PDO::PARAM_INT);
-$stmt->execute();
+require_once("MsCommon_mid-refactor.inc");
 
-$query = 'SELECT exist FROM #TESTTABLE';
+try {
+    $conn = connect();
+    $sample = 1234567890.1234;
 
-//prepare with no buffered cursor
-print "no buffered cursor, stringify off, fetch_numeric off\n"; //stringify and fetch_numeric is off by default
-$stmt = $conn->prepare($query);
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch( PDO::FETCH_BOUND );
-var_dump ($decimal_col);
+    $tbname = "TESTTABLE";
+    createTable($conn, $tbname, array("exist" => "decimal(18,8)"));
```

Comment:
I actually don't like the column name `exist`, and in such case we could have used ColumnMeta to take care of creating a default column name
in fetching data below we could have just used `select * `

Reply:
haha I feel the same when I was refactoring. I didn't want to have to think of a new column name so I left it. But if you feel really strongly about it I can change it.

Reply:
yes please change this odd column name to something less strange